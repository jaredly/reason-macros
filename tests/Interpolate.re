let%macro simple = (num: int) => "hi $eval{num} times";
let%macro more = (str: string, num: int) => "hi $eval{num} and $eval{str}";

[%simple 5];
[%more ("People", 3)];