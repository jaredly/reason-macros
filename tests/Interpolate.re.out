"hi 5 times";
"hi 3 and People";
